pkg/api/network/store.go

```diff
@@ -0,0 +1,76 @@
+package network
+
+import (
+	"encoding/json"
+	"errors"
+	"strconv"
+
+	cniv1 "github.com/containernetworking/cni/pkg/types"
+	"github.com/rancher/apiserver/pkg/apierror"
+	"github.com/rancher/apiserver/pkg/types"
+	cni "github.com/rancher/harvester/pkg/generated/controllers/k8s.cni.cncf.io/v1"
+	"github.com/rancher/wrangler/pkg/schemas/validation"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+type networkStore struct {
+	types.Store
+	nadCache cni.NetworkAttachmentDefinitionCache
+}
+
+type NetConf struct {
+	cniv1.NetConf
+	BrName       string `json:"bridge"`
+	IsGW         bool   `json:"isGateway"`
+	IsDefaultGW  bool   `json:"isDefaultGateway"`
```

Comment:
Can we make these fields consistent with the JSON tags

Reply:
this is intended to be consistent with the origin bridge NetConf at https://github.com/containernetworking/plugins/blob/master/plugins/main/bridge/bridge.go#L47, and since it is the main package so we had to do a hard-copy in here.